scripts/time-filter-proto.js

```diff
@@ -43,9 +43,11 @@ export function setup () {
   const transportation = __ENV.TRANSPORTATION || 'driving+ferry'
   const protocol = __ENV.PROTOCOL || 'https'
   const travelTime = parseInt(__ENV.TRAVEL_TIME || 7200)
-  const envCountry = __ENV.COUNTRY
-  const countryCoords = getProtoCountryCoordinates(envCountry, __ENV.COORDINATES, true)
-  const country = envCountry || 'uk'
+
+  const location = __ENV.LOCATION || 'UK/London'
+  const country = location.slice(0, 2).toLowerCase()
+  const countryCoords = getProtoCountryCoordinates(location)
+
   const query = __ENV.QUERY || `api/v2/${countryCode(country)}/time-filter/fast/${transportation}`
```

Comment:
the `location.slice` is used to get the iso2 country-code from the location, it's needed to construct the proto request url. I could make the user specify the iso2 code manually, but this might be simpler

Reply:
tho it could break if the user decides to add a new location and doesn't follow the same naming pattern